test/d2l-icon-button.html

```diff
@@ -0,0 +1,54 @@
+<!doctype html>
+<html>
+	<head>
+		<meta charset="UTF-8">
+		<title>d2l-icon-button tests</title>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+		<script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+		<script src="../../web-component-tester/browser.js"></script>
+		<link rel="import" href="../../iron-test-helpers/iron-test-helpers.html">
+		<link rel="import" href="../d2l-icons.html">
+	</head>
+	<body>
+		<test-fixture id="d2l-icon-button">
+			<template>
+				<button is="d2l-icon-button" icon="d2l-tier1:chevron-right">Chevron Right</button>
+			</template>
+		</test-fixture>
+
+		<test-fixture id="trim-whitespaces">
+			<template>
+				<button is="d2l-icon-button" icon="d2l-tier1:chevron-right"> Chevron Right </button>
+			</template>
+		</test-fixture>
+
+		<script>
+			describe('<d2l-icon-button>', function() {
+				describe('standard tests', function(){
+					beforeEach(function(done) {
+						iconButton = fixture('d2l-icon-button');
+						flush(done);
+					});
+
+					it('should set icon correctly', function() {
+						expect(iconButton.icon).to.be.equal("d2l-tier1:chevron-right");
+					});
+
+					it('should set title with text content', function() {
+						expect(iconButton.title).to.be.equal("Chevron Right");
+					});
+				});
+				describe('special cases', function() {
+					beforeEach(function(done) {
+						iconButton = fixture('trim-whitespaces');
+						flush(done);
+					});
+					it('should trim whitespaces from text content while displaying title', function() {
+						expect(iconButton.title).to.be.equal("Chevron Right");
+						expect(iconButton.title).to.not.be.equal(" Chevron Right ");
```

Comment:
I agree with Anthony here, the second assertion isn't needed and is just extra code that's not testing anything new. It would be like testing if something is `true` and then testing to make sure it's not `false` -- obviously it's not `false` because you know it's `true`. :)